### DISCUSSION TITLE
Are you having indexing issues for your landing pages

### DISCUSSION THREAD

**Discussion by promotionsonly** [YOURLS/YOURLS#2882] [General]
Hello, I was wondering to those using the YOURLS platform if you are receiving Google Search Console indexing issues for your landing page domain (not your shorturl domain)

**URL is not on Google: Indexing Issues caused by the X-Robots-Tag.** 
The issue is GSC reports that your landing page cannot be indexed because its found a X-Robots-Tag on your landing page, but in actual fact this tag comes from the refferer page that is displayed (my yourls domain). I posted on asking [reddit/r/techseo](https://www.reddit.com/r/TechSEO/comments/maf4m2/url_is_not_on_google_indexing_issue_xrobotstag/) for the community for help but i also want to know if anyone else using the yourls platform are receiving these messages

Miguel

**Comment by ozh** [comment]
Not sure what you're calling the "landing page domain". Is that the "long URL that the short URL is pointing to" ?

On redirecting a short URL, YOURLS does [add a X-Robots-Tag](https://github.com/YOURLS/YOURLS/blob/af30597777b6a51f8cc0c2d75e9b32bace07fe65/includes/functions.php#L255-L258) to prevent Googlebot from indexing short URLs (ie Googlebot should index longurl.com/this, longurl.com/that and so on but not the short URLs sho.rt/a and sho.rt/b pointing to these destinations)

If I'm correct that shouldn't impact how your site is indexed, unless of course the only links ever pointing to your site that the googlebot sees are short URLs...

**Reply by promotionsonly** [reply]
Yes what i mean by the "landing page domain" is the final destination url (the longurl). I was just wondering if anyone is receiving the same GSC indexing errors as I am for their longurl domain where the longurl page cannot be indexed because its found a x-robots-tag. For some reason GSC thinks the x-robots-tag is on the longurl. Its not an issue with the YOURLS platform, it might be an issue with googlebot following a 301 + x-robots-tag redirect. https://www.reddit.com/r/TechSEO/comments/maf4m2/url_is_not_on_google_indexing_issue_xrobotstag/
